tests/finite_element_problem/test_linear_simple_beam.py

```diff
@@ -0,0 +1,64 @@
+import os
+import pytest
+from pathlib import Path
+
+from fenicsxconcrete.helper import Parameters
+from fenicsxconcrete.unit_registry import ureg
+from fenicsxconcrete.experimental_setup.simple_beam import SimpleBeam
+from fenicsxconcrete.finite_element_problem.linear_elasticity import LinearElasticity
+from fenicsxconcrete.sensor_definition.displacement_sensor import DisplacementSensor
+
+
+@pytest.mark.parametrize("dimension", [[2, [0.00105057, -0.01310806]],
+                                       [3, [1.13946512e-03,  1.42368783e-05, -1.42250453e-02]]])
+def test_linear_simple_beam(dimension):
+    dim = dimension[0]
+    result = dimension[1]
```

Comment:
For me the naming is missleading (dimension =/= results), but I saw that this notation is used in all test ...
maybe something like: "dimension, results", [[],[]] could work

Reply:
I agree, but could not think of a better way.
I will have a quick look if this can be improved, but I am not too worried in this case about it being misleading as it is just for setting up our tests.

Reply:
it works!